libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Analytics/SFSDKSalesforceAnalyticsManager.m

```diff
@@ -131,16 +131,25 @@ - (instancetype) initWithUser:(SFUserAccount *) userAccount {
         DataDecryptorBlock dataDecryptorBlock = ^NSData*(NSData *data) {
             return [SFSDKCryptoUtils aes256DecryptData:data withKey:encKey.key iv:encKey.initializationVector];
         };
-        self.analyticsManager = [[SFSDKAnalyticsManager alloc] initWithStoreDirectory:rootStoreDir dataEncryptorBlock:dataEncryptorBlock dataDecryptorBlock:dataDecryptorBlock deviceAttributes:deviceAttributes];
-        self.eventStoreManager = self.analyticsManager.storeManager;
-        self.remotes = [[NSMutableArray alloc] init];
-        SFSDKAnalyticsTransformPublisherPair *tpp = [[SFSDKAnalyticsTransformPublisherPair alloc] initWithTransform:[[SFSDKAILTNTransform alloc] init] publisher:[[SFSDKAILTNPublisher alloc] init]];
-        [self.remotes addObject:tpp];
+        _analyticsManager = [[SFSDKAnalyticsManager alloc] initWithStoreDirectory:rootStoreDir dataEncryptorBlock:dataEncryptorBlock dataDecryptorBlock:dataDecryptorBlock deviceAttributes:deviceAttributes];
+        _eventStoreManager = self.analyticsManager.storeManager;
+        _remotes = [[NSMutableArray alloc] init];
+        
+        // There's no standard for unauthenticated instrumentation publishing, currently.  Consumers
```

Comment:
Comment pretty much covers it, but just a heads-up.  This will likely be different for different consumer scenarios.